homeassistant/components/anova_cooker/sensor.py

```diff
@@ -0,0 +1,85 @@
+"""Anova sous vide cooker integration."""
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import Entity
+
+
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Add sensors for passed config_entry in HA."""
+    async_add_devices([AnovaCookerSensor(config_entry.data["device_id"])])
+
+
+class AnovaCookerSensor(Entity):
+    """Anova Cooker Sensor."""
+
+    def __init__(self, device_id):
+        """Set up the device_id, cooker, and empty state & attr."""
```

Comment:
I would expect different sensor types for my anova, like remaining time, current temperature, etc. so setting a `device_class` would be my advice, even if it is just one at this time.